core/sql/arkcmp/CompException.cpp

```diff
@@ -113,11 +114,21 @@ FatalException::FatalException(const char *msg,
   else {
     msg_[0] = 0;
   }
+  if(stackTrace) {
+    strncpy(stackTrace_, stackTrace, sizeof(stackTrace_));
+    stackTrace_[sizeof(stackTrace_)-1] = 0;
```

Comment:
Thanks for the review.  Have fixed to address the issue.  sizeof(stackTrace_) should be ok here since it will be receiving buffer size.